### ISSUE TITLE
Accessibility: Audit

### ISSUE THREAD

**Issue by stephanie-roe** [whiteheadol/whats-cookin#94]
[ ] You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
